File: src/test/java/com/juan/StudentTest.java
package com.juan;

import com.juan.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void hightestScoreTest(){
        Student student = new Student("Yuan" , 60 , 80);
        Assertions.assertEquals(80 ,student.highest());
    }

    @Test
    public void averageTest() {
        Student student = new Student("Yuan", 60, 80);
        Assertions.assertEquals((80+60)/2, student.getAverage());
    }
}

File: src/test/kotlin/com/juan/StudentTest.kt
package com.juan.kotlin

import com.juan.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {
    @Test
    fun hightestTest() {
        val student = Student("Yuan",60 , 80)
        Assertions.assertEquals(80,student.highest())
    }
    @Test
    fun averageTest() {
        val student = Student("Yuan",60 , 80)
        Assertions.assertEquals((60+80)/2,student.getAverage())
    }
    @Test
    fun gradingTest() {
        val student = Student("Yuan",60 , 80)
        Assertions.assertEquals('C',student.grading())
    }
}

